layers/core_validation.cpp

```diff
@@ -5172,13 +5294,15 @@ bool CoreChecks::PreCallValidateGetPhysicalDeviceImageFormatProperties2(VkPhysic
     return skip;
 }
 
+#if !defined(VULKANSC) // VK_KHR_get_physical_device_properties2 is promoted to core in VulkanSC
```

Comment:
nit: these are also promoted to core in Vulkan, maybe these comments should say "is only available as core functionality in Vulkan SC"?

Reply:
Agreed, fixing in the next patch.

Reply:
Maybe I just should just replace these with the extension defines. I'm not sure why I used VULKANSC here